spec/html/proofer/fixtures/images/relativeWithBase.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <base href="/links/">
```

Comment:
It seems like the `base` calculation is relative to the location of the file being tested:

```
"html-proofer/spec/html/proofer/fixtures/images/links/gpl.png"
"html-proofer/spec/html/proofer/fixtures/images/links/folder/assets/barrel.png"
"html-proofer/spec/html/proofer/fixtures/images/images/gpl.png"
"html-proofer/spec/html/proofer/fixtures/images/images/gpl.png"
```

I don't know if that's correct or not, but it's definitely why the test is failing here.